### PR TITLE
Refactor `pack` command to support `-o`/`--output` flag for custom OC…

### PR DESCRIPTION
**Pull request by hbelmiro**
…I layout directories; update default output to `<project>/build/`.

### CODE REVIEW

File: internal/cli/pack_test.go [hunk 1]
```diff
@@ -75,14 +80,204 @@ func TestPack_WithManifestFlagFromOtherDir(t *testing.T) {
...
+	data, err := json.Marshal(manifest)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err := os.WriteFile(filepath.Join(projectDir, "artifact.json"), data, 0o600); err != nil {
+		t.Fatal(err)
+	}
+	if err := os.WriteFile(filepath.Join(projectDir, "SKILL.md"), []byte("# x"), 0o600); err != nil {
+		t.Fatal(err)
+	}
```
[the remark this answers is not in the document]
Comment (hbelmiro) [reply internal/cli/pack_test.go]: Done. Extracted `setupTestProject` helper — all 6 test functions now delegate manifest creation + file writing to it.

File: internal/cli/pack.go [hunk 2]
```diff
@@ -31,7 +38,15 @@ func newPackCmd() *cobra.Command {
 			if err != nil {
 				return fmt.Errorf("load manifest: %w", err)
 			}
-			layoutPath := filepath.Join(projectRoot, defaultLayoutDir)
+			var layoutPath string
+			if trimmed := strings.TrimSpace(outputFlag); trimmed == "" {
+				layoutPath = filepath.Join(projectRoot, defaultLayoutDir)
+			} else {
+				layoutPath, err = filepath.Abs(filepath.Clean(trimmed))
+				if err != nil {
```
[the remark this answers is not in the document]
Comment (hbelmiro) [reply internal/cli/pack.go:46]: Done. Aligned `pull --output` with `pack --output`: now trims whitespace-only values (falls back to default) and applies `filepath.Clean` + absolute resolution for relative paths. Added test for the whitespace edge case.
